nf1_cellprofiler_classifier_fs_all_cellprofiler_all_feat_perfect.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>I recommend adding a title for the whole notebook with a quick description of what this notebook does or analyzes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/NF1_SchwannCell_data_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Agreed